packages/aws/data_stream/config/fields/ecs.yml

```
@@ -0,0 +1,4 @@
+- name: cloud.provider
+  type: constant_keyword
+- name: observer.vendor
+  type: constant_keyword
```

If its defined this way, the fields inside README is rendered without description.
```
| cloud.provider |  | constant_keyword |
| observer.vendor |  | constant_keyword |
```
Please update to:
```suggestion
- name: cloud.provider
  type: constant_keyword
  external: ecs
- name: observer.vendor
  type: constant_keyword
  external: ecs
```